target/linux/ramips/dts/mt7621_hilink_hlk-7621a.dts

```diff
@@ -0,0 +1,88 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+/ {
+	compatible = "hlk,mt7621a-module", "mediatek,mt7621-soc";
+	model = "HLK MT7621A Module";
```

Comment:
Please use the proper name and compatible here, matching what you specify in the Makefile below (just with underscore -> comma).